pkg/apis/sources/v1alpha1/gitlabsource_lifecyle.go

```diff
@@ -0,0 +1,128 @@
+/*
+Copyright 2020 The Knative Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"sort"
+	"strings"
+)
+
+// String prepended to GitLab event types to make them fully-qualified.
+const eventPrefixGitLab = "dev.knative.sources.gitlab."
+
+// Types of events emitted by a GitLabSource.
+// The chosen format an case matches the "object_kind" attribute contained in
```

Comment:
"and"

Reply:
Fixed in [7cd3e48f..ebef4c6e](https://github.com/knative-sandbox/eventing-gitlab/compare/7cd3e48f70ea9ae668b7422645af0b731e99480f..ebef4c6e4bfaf5f8caa6ada06c724164be119f18)